pydra/engine/node.py

```diff
@@ -1,19 +1,38 @@
-"""Basic compute graph elements"""
+"""Basic compute graph elements."""
```

Comment:
This isn't really a sentence, more of a header. I don't think we need punctuation.

We will probably want to expand the docs at some point.